homeassistant/components/smartthings/.translations/en.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "title": "SmartThings",
+    "step": {
+      "user": {
+        "title": "Enter Personal Access Token",
+        "description": "Please enter a SmartThings [Personal Access Token](https://account.smartthings.com/tokens) that has been created per the [instructions](https://www.home-assistant.io/components/smartthings/).",
```

Comment:
We should use placeholders and pass the url from Python, that way translators can't mess with it, and if we need to change the url, the translators don't have to update the string.

Reply:
Addressed in latest push!  Thanks!